src/main/java/com/mixpanel/android/mpmetrics/MixpanelFCMMessagingService.java

```diff
@@ -243,13 +255,24 @@ protected static Notification buildNotification(Context context, Intent inboundI
         maybeSetCustomIconColor(builder, data);
         maybeAddActionButtons(builder, data, context);
         maybeSetChannel(builder, data, context);
+        maybeSetSubTitle(builder, data, context);
+        maybeSetExpandableImage(builder, data, context);
```

Comment:
this is just going to slam the changes of `maybeSetExpandableNotification`. it seems to me that we should handle this inside of `maybeSetExpandableNotification`.